source/content/guides/support/02-contact-support.md

```diff
@@ -48,15 +46,15 @@ The ticket support feature is available to certain Account packages and account
 
 After a ticket is submitted, you can view details for your support requests. If are you are part of an Workspace, your support tickets are visible to all members except [Unprivileged users](/guides/account-mgmt/workspace-sites-teams/teams#organizations-roles-and-permissions).
 
-## Real Time Chat Support
+## Live Chat
 
 Start a chat with our Support Team to ask questions or request assistance on issues within our [scope of support](#scope-of-support). This support feature is available to all users and sites across all plans, including Sandbox.
 
 To receive the highest level of support available to your site or Workspace, remember to access the chat from the Workspace if available.
 
 1. [Go to the workspace](/guides/account-mgmt/workspace-sites-teams/workspaces#switch-between-workspaces) or [Site Dashboard](/guides/account-mgmt/workspace-sites-teams/sites#site-dashboard) affected by the issue.
 
-1. Click **Support**, **Launch Chat**, then **New Conversation**.
+1. Go to the **Support** tab, then select **Start chat**.
```

Comment:
Phoebe dashboard currently says, "start a chat", are we shortening this in the UI?

Reply:
Right now it says 'start a chat' and it will say that with this round of changes. Ingrid has requested we shorten to 'start chat' in the near future. I am OK using 'Start chat' as I think this is very similar to what will show in the dashboard now until we make the next update to the experience.